concrete/src/Package/Package.php

```diff
@@ -631,28 +641,22 @@ public function getPackageEntityPaths()
     public function installDatabase()
     {
         $this->installEntitiesDatabase();
-
         static::installDB($this->getPackagePath() . '/' . FILENAME_PACKAGE_DB);
     }
 
     public function installEntitiesDatabase()
```

Comment:
Even just having that as part of the packageservice would make me very happy. I had a plan at some point to do the following:

* Create package install routine builder like `$installer->install($installer->singlePage(...), $installer->theme(...))` or something like that
* Make a new interface for routine installation like `SelfInstallableInterface` with a method on it `->getRoutine(): InstallRoutine`
* Make a package installer that will detect instances of `SelfInstallableInterface` and do something like `$package->getRoutine()->getQueue()->execute();` to install